neurodamus/cell_distributor.py

```diff
@@ -154,8 +153,8 @@ def __init__(self, circuit_conf, target_manager, _run_conf=None, **_kw):
     pc = property(lambda self: self._pc)
     population_name = property(lambda self: self._population_name)
     circuit_target = property(lambda self: self._target_spec.name)
-    circuit_name = property(lambda self: self._circuit_name)
-    is_default = property(lambda self: self._circuit_name is None)
+    circuit_name = property(lambda self: self._circuit_conf.name)
+    is_default = property(lambda self: self.circuit_name is None)
```

Comment:
BTW, `is_default = True` is for BlueConfig circuit. In SONATA case, the circuit for each population is in `extra_circuits` with a real circuit_name. At some point, we can clean this logic, remove `is_default`. Needs to check where it is used.

Reply:
Great, I won't do it in this PR, but for future reference, I added a ticket: https://github.com/openbraininstitute/neurodamus/issues/118